config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
```

Comment:
Well done adding controller actions as requested by the previous reviewer 👍. However, your routes are not properly connected.

We need to modify the routes to get access to the corresponding views 😊🛠

e.g
```ruby
get '/patients/:id', to: 'patients#show'
```
This asks the router `/patients/:id` to match it to a controller action. 

The Rails router recognizes URLs and dispatches them to a controller's action, or a Rack application. It can also generate paths and URLs, avoiding the need to hardcode strings in your views.


To confirm everything works as expected, typing the command `rails routes | grep user` should yield URLs that correspond to these wireframes 

![image-1](https://github.com/firstpj/blog-app/assets/78251927/58f3457d-be68-426b-b471-ffb4cb81c8a5)

Your result
![image](https://github.com/firstpj/blog-app/assets/78251927/1b80ed66-26fb-4c17-9d48-731604656413)

Expected
![image](https://github.com/firstpj/blog-app/assets/78251927/e7802df7-af20-4287-ac60-bdbe021f7f2d)

kindly click here to [learn](https://guides.rubyonrails.org/routing.html#dynamic-segments) how to fix this🤗 👍

Reply:
Also, please ensure that `posts_controller` has `#index` and `#show` actions. 🙏🤗🛠

e.g
```ruby
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```